lib/jelix-modules/jacl2db/daos/jacl2rights.dao.xml

```diff
@@ -26,6 +26,14 @@
                 <in property="id_aclgrp" expr="$groups" />
             </conditions>
         </method>
+        <method name="getManageGroupRightsByGroups" type="select">
```

Comment:
It returns all rights for the given resource and groups, regardless of the subject ! But for example, a resource value "1", could not mean the same thing for a subject "news.update" and for a subject "user.update" (since in this case "1" may refers to a news id for "news.update", and a user id for the other. I don't see the use case of this method (and probably its name does not really match the behavior).

I think the subject should be a parameter, and in this case, there is already such method : getRightWithRes()